### PR TITLE
Fixes documentation of v_scroll_speed and wrap_enabled

### LINKED ISSUE #25707: Editor: TextEdit v_scroll_speed tooltip error
none found

**Godot version:**

3.1 beta 3

**OS/device including version:**
Windows 10

**Issue description:**
v_scroll_speed's tooltip displays wrong text
immagine

**Steps to reproduce:**
point mouse on V_Scroll_Speed 

**Minimal reproduction project:**
-

### PR DESCRIPTION
**Pull request by siddharth952**
Fixed #25707 
Modified the documentation regarding v_scroll_speed and wrap_enabled.

### CODE REVIEW

File: doc/classes/TextEdit.xml [hunk 1]
```diff
@@ -401,9 +401,10 @@
 			String value of the [TextEdit].
 		</member>
 		<member name="v_scroll_speed" type="float" setter="set_v_scroll_speed" getter="get_v_scroll_speed">
-			If [code]true[/code], enables text wrapping when it goes beyond he edge of what is visible.
+			Vertical scroll sensitivity.
 		</member>
 		<member name="wrap_enabled" type="bool" setter="set_wrap_enabled" getter="is_wrap_enabled">
+			If [code]true[/code], enables text wrapping when it goes beyond he edge of what is visible.
```
Comment (asheraryam) [inline doc/classes/TextEdit.xml]: Might be worth fixing the typo from "beyond **he** edge" to "beyond **the** edge".

Comment (asheraryam) [inline doc/classes/TextEdit.xml]: Might be worth fixing the typo from "beyond **he** edge" to "beyond **the** edge".

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thank you!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.1.

**Pull request merged**